test/DynamoCoreTests/DynamoCoreTests.csproj

```diff
@@ -149,18 +133,6 @@
     <Copy SourceFiles="$(PkgMicrosoft_Diagnostics_Runtime)\lib\netstandard2.0\Microsoft.Diagnostics.Runtime.dll" DestinationFolder="$(TestDependenciesPath)" />
 </Target>
 
-  <Target Name="MoveSomeTestDepsOutOfBin_net48" AfterTargets="Build" Condition=" '$(TargetFramework)' == 'net48' ">
```

Comment:
I am not 100% sure about the purpose of this.. @pinzart90 and you would know better than me 👍🏻

Reply:
The reason I removed it is as we are not targeting net48 in this branch anymore.